vm/2.0/modules/ROOT/pages/vm-studio-configuration.adoc

```diff
@@ -0,0 +1,137 @@
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-input-source,Configure a source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+
+When you run the connector, you can view the app log to check for problems, as described in <<view-app-log,View the App Log>>.
+
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:vm-reference.adoc[VM Connector Reference].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add <connector name> to your Mule project to automatically populate the XML code with the connector's namespace and schema location and to add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In the *Add Dependencies to Project* window, type `vm connector` in the search field.
+. Click *VM Connector* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+
+[[configure-input-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these input sources to use with <connector name>:
+
+* *VM Listener*, which initiates a flow by listening on a VM queue.
+* *HTTP Listener*, which initiates a flow each time it receives a request on the configured host and port
+* *Scheduler*, which initiates a flow when a time-based condition is met
+
+For example, to configure a *VM Listener* source, follow these steps:
+
+. In the *Mule Palette* view, select *VM > Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Click the plus sign (+) next to the *Connector configuration* field to configure a VM queue that can be used by all instances of the *Listener* source and other VM Connector operations in the app.
+. <<configure-global-element,Define the VQ queue>>
+. On the *Listener* configuration screen, set the *Number of consumers* field to your desired value. Defaults to `4`. +
+. In the *Queue* section, select a *Queue name* from the drop-down box. +
+If you previously created a VM queue, the *Queue name* value appears automatically.
+. Set the *Timeout* to your desired value. Defaults to `5`.
+. Select a *Timeout unit* from the drop-down box.
```

Comment:
```suggestion
. Select a *Timeout unit* from the drop-down menu.
```
Does the template say "box"? Because we typically select from a menu, and "drop-down" typically precedes "menu."